network-api/networkapi/wagtailpages/pagemodels/research_hub/relations.py

```diff
@@ -21,7 +21,7 @@ class ResearchAuthorRelation(wagtail_models.TranslatableMixin, wagtail_models.Or
     )
 
     panels = [
-        snippet_handlers.SnippetChooserPanel("author_profile"),
+        snippet_handlers.FieldPanel("author_profile"),
```

Comment:
I am not sure if that is possible, but since we now use the `FieldPanel` can we change the import to `wagtail.admin import panels` and use `panels.FieldPanel`?